pkg/ccl/cloudccl/amazon/s3_connection_test.go

```diff
@@ -11,6 +11,7 @@ package amazon
 import (
 	"context"
 	"fmt"
+	"github.com/cockroachdb/cockroach/pkg/cloud/cloudtestutils"
```

Comment:
the linter will probs complain about this

Reply:
fixed